src/clients/nominatimClient.ts

```
@@ -0,0 +1,28 @@
+import type { GeocodeParams } from '../types/geocodeTypes.js'
+
+const nominatimClient = axios.create({
+  baseURL: 'https://nominatim.geocoding.ai/',
+  headers: {
+    'User-Agent': 'GeocodingMCP/1.0',
+  },
+})
+
+const parseResult = (result: any[]) => {
+  result.forEach((item: any) => {
+    delete item.licence
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)